declaration/SpinViewer.d.ts

```diff
@@ -1,5 +1,3 @@
-import Component from "@egjs/component";
```

Comment:
@WoodNeck 

Maybe
```js
import Component from "@egjs/component";

class SpinViewer extends Component {}
```

Reply:
Oh, you're right.
I've shouldn't remove that. Thanks for checking.